cmd/common.go

```diff
@@ -446,3 +447,47 @@ func connectSSH(ctx context.Context, sshUser, sshIP string, sshPort int, key str
 
 	return client, nil
 }
+
+func copyPastelUpToRemote(ctx context.Context, client *utils.Client, remotePastelUp string) error {
+	// Check if the current os is linux
+	if runtime.GOOS == "linux" {
+		log.WithContext(ctx).Infof("copying pastelup to remote")
```

Comment:
nit: could you unify your naming rule? Like log message should start with Upper case or lower case?
like below:
```
log.WithContext(ctx).Infof("Current OS is not linux, skipping pastelup copy")
```